tools/src/codespell/requirements.txt

```diff
@@ -0,0 +1 @@
+codespell==2.1.0
```

Comment:
Could we pin hash for all python dependences as Envoy does?

Reply:
We could, but the tooling to do this isn't great. Well, I mean https://pypi.org/project/pip-tools/ is great, but the exact dependency resolution is pretty sensitive to host OS and specific Python version.  Because everything is coming from PyPI (not random servers scattered around the net), and PyPI is pretty good about not letting you overwrite an existing version, IMO the benefit would be minimal.

If we had lots of Python code, then yeah, it'd be worth it. But just installing a tool or two?

Reply:
Sounds reasonable to me :)